src/commands/utility/serverinfo.js

```diff
@@ -0,0 +1,40 @@
+const Command = require('../../Command')
+const Discord = require('discord.js');
+const util = require('../../util');
+
+class ServerinfoCommand extends Command{
+  
+    static names = ['si', 'server'];
+
+    static usage = '';
+
+    static description = 'Show the servers info';
+
+    async execute() {
+        const guild = this.message.guild;
+        
+        let generic = '';
+        generic += `**Owner:** ${guild.owner.user.username} \n`;
+        generic += `**Owner ID:** ${guild.ownerID} \n`;
+        generic += `**Created:** ${guild.createdAt.toDateString()} \n`;
+        generic += `**Region:** ${guild.region} \n`;
```

Comment:
The region is fully lowercase which looks a bit odd.
Maybe convert it to upper case?